check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
```

Comment:
Prohibits complex (deeply nested) type annotations (configured at the plugin's default level).